### PR TITLE
Add option to control shuffling of training data

### PR DESCRIPTION
**Pull request by aakashns**
Currently, the training the data in `ImageClassifierData` is always shuffled randomly using [`RandomSampler`](http://pytorch.org/docs/master/data.html#torch.utils.data.sampler.RandomSampler), and there is no way to control this behavior. This change introduces a new argument `shuf_trn` which allows the user to turn off shuffling.

This option can be useful in the following situations:
* You've already shuffled the CSV file and you want to train with the same deterministic set of batches in every epoch
* You want the predictions for training data in the right order (i.e. matching the rows in the CSV)
* You want to compute the activations of some intermediate layer and save them to disk

In all of the above cases, it helps to process the training data without shuffling.

### CODE REVIEW

**Comment by jph00** [comment]
I think you should just be able to use fix_dl in this case?

**Comment by aakashns** [comment]
Shit, I totally missed that! Thanks for pointing it out.
